xgcm/test/test_derivatives.py

```diff
@@ -1,30 +1,135 @@
 from __future__ import print_function
 import xarray as xr
 
+import numpy as np
+
 from xgcm.grid import Grid, Axis
+from xgcm.test.datasets import datasets_grid_metric
 
 
 def test_derivative_uniform_grid():
```

Comment:
Do we still need this test function? Or is it superseded by the ones implemented below? @rabernat